Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -281,10 +281,19 @@ struct Scene_c3t3_item_priv {
   }
   ~Scene_c3t3_item_priv()
   {
-     delete frame;
-    delete tet_Slider;
   }
-
+  void destroy()
+  {
+    c3t3.clear();
+    tree.clear();
+    if(frame)
+    {
+      static_cast<CGAL::Three::Viewer_interface*>(QGLViewer::QGLViewerPool().first())->setManipulatedFrame(0);
+      delete frame;
+      frame = NULL;
+      delete tet_Slider;
+    }
+  }
```

Comment:
I do no understand why there is a function `Scene_c3t3_item_priv::destroy()`. Why is not it merged with `~Scene_c3t3_priv()`?